### PR TITLE
Feature/task 992 create sequencing con

### PR DESCRIPTION
**Pull request by maurpa**
Concept for modelling Sequences:
- Adds Mapping of Interface End to transition triggers
- Adds Mapping of Target state to transition trigger (cross different state machines)

### CODE REVIEW

File: de.dlr.sc.virsat.product/virsat_core.product [hunk 1]
```diff
@@ -142,7 +143,6 @@ Germany
       <feature id="org.eclipse.rmf.reqif10.feature" installMode="root"/>
       <feature id="org.eclipse.rmf.reqif10.pror.feature" installMode="root"/>
       <feature id="org.eclipse.rmf.reqif10.thirdparty.feature" installMode="root"/>
-      <feature id="de.dlr.sc.virsat.model.extension.thermal.feature" installMode="root"/>
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.product/virsat_core.product:145]: Removal of thermal doesn't look right

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Looks good 👍 Only one issue that stoof out to me.

File: de.dlr.sc.virsat.product/virsat_core.product (hunk 1 above)
Comment (maurpa) [reply de.dlr.sc.virsat.product/virsat_core.product:145]: I removed it because it is included two times in the product file (line 145 and line 90)

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

File: de.dlr.sc.virsat.model.extension.sequencing.test/.settings/org.eclipse.m2e.core.prefs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
[the remark this answers is not in the document]
Comment (maurpa) [reply de.dlr.sc.virsat.model.extension.sequencing.test/.settings/org.eclipse.m2e.core.prefs:1]: The settings are identical with the ones of other features. For example compare the settings of thermal.feature , power.budget.feature

**Review by GeorgeNasrDLR — APPROVED** [review]

**Pull request merged**